### PR TITLE
Objectgroups in Tiles (fixed)

### PR DESCRIPTION
**Pull request by TheUltimateHose**
So this time, everything is fixed and actually tested: The tilemap demo
works just fine (apart from Game.Exit() being obsolete). Those
objectgroups are now available in their respective TileSetTiles as seen
in this screenshot: http://screencast.com/t/YvdCmQoUcd

### CODE REVIEW

File: Source/MonoGame.Extended/Maps/Tiled/TiledMapReader.cs [hunk 1]
```diff
@@ -52,6 +53,8 @@ protected override TiledMap Read(ContentReader reader, TiledMap existingInstance
                         tileSetTile.CreateTileSetTileFrame(order: k, tileId: frameId, duration: reader.ReadInt32());
                     }
                     ReadCustomProperties(reader, tileSetTile.Properties);
+                    
+                    tileSetTile.ObjectGroups.AddRange(ReadObjectGroups(reader));
```
Comment (craftworkgames) [inline Source/MonoGame.Extended/Maps/Tiled/TiledMapReader.cs]: Are you sure you didn't miss a file when committing? I'm getting a compile error on this line.

**Comment by craftworkgames** [comment]
I'm getting the following compile error:

> Error	CS1061	'TiledTileSetTile' does not contain a definition for 'ObjectGroups' and no extension method 'ObjectGroups' accepting a first argument of type 'TiledTileSetTile' could be found (are you missing a using directive or an assembly reference?)	MonoGame.Extended	D:\Github\Public\MonoGame.Extended-249\Source\MonoGame.Extended\Maps\Tiled\TiledMapReader.cs	57	Active

Sorry mate.

**Comment by TheUltimateHose** [comment]
Jep, I missed a file. I backed up most files when I merged the remote-tracking branch, but somehow, I missed that one, so here it is.

**Comment by craftworkgames** [comment]
Great work. Thanks :)

**Comment by OptimusPi** [comment]
Good job, keep up the good work. I ran the Platformer effortlessly last night, it's really exciting, and I am happy for the work going into it especially the Tiled Map aspect. Keep up the great work

**Pull request merged**